.circleci/smoketest_k8s.sh

```diff
@@ -161,7 +165,7 @@ echo ""
 echo "=================================================="
 echo "Starting verifier"
 echo "=================================================="
-kubectl run -it --restart=Never ${contname_verifier} --image=${smoketest_image} -- \
+kubectl run -i --restart=Never ${contname_verifier} --image=${smoketest_image} -- \
```

Comment:
Any special reason why ``-t`` option was removed and we don't want / need tty anymore?

Reply:
For some reason, it couldn't run the container on GitHub actions when I tried to run old smoke tests there. For now, I just left them on circleci without moving them here, because there are many things to change in order to make them work on GH actions.